src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -30,7 +30,7 @@ import scala.annotation.tailrec
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.concurrent.duration._
-import scala.language.postfixOps
+import scala.language.{implicitConversions, postfixOps}
```

Comment:
Ah nothing, I added an `implicit def` which would have unleashed hell on me but removed it in the end - the import stayed